ansible/roles/worker/templates/girder_worker.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Girder Worker Service
+After=network.target
+
+[Service]
+Type=simple
+User={{ worker_exec_user }}
+Group={{ worker_exec_group }}
+WorkingDirectory=/opt/girder_worker
+ExecStart=/bin/sh -c "LANG=en_US.UTF-8 python -m girder_worker --concurrency={{worker_concurrency}} --pidfile=.worker_pid --logfile=/opt/girder_worker/worker.log -l info"
```

Comment:
I think it probably should be in both Girder and Girder Worker roles, at least when they are run in docker containers.